README.md

```diff
@@ -24,6 +24,9 @@ These is a single route, `/article/:id`, where `id` is the filename of the stati
 
 **NOTE:** the `id` parameter is currently redundant as the Article component is hard-coded to fetch data from [scenario-01](https://github.com/bbc/simorgh/blob/latest/data/test/scenario-01.json). Furthermore, the Article component does not do anything with that data.
 
+### View a breakdown of the bundle size with webpack-bundle-analyzer
+With dev server running go to [http://127.0.0.1:8888/](http://127.0.0.1:8888/)
```

Comment:
You should run the bundle analyser on the production build - this is so that you get an accurate picture of the dependencies in the JS bundle (not including dev dependencies).